int32 center_x
int32 center_y

int32 height
int32 width